.golangci.next.reference.yml

```diff
@@ -3493,6 +3493,30 @@ linters-settings:
     # Default: false
     constant-kind: true
 
```

Comment:
I guess it's necessary to add `usetesting` to the `linters.enable` and `linters.disable` lists.